README.md

```diff
@@ -359,6 +359,14 @@ end_gcode:
 #   A list of G-Code commands to execute after each calibration command.
 #   See docs/Command_Templates.md for G-Code format. This can be used to
 #   detach the probe afterwards.
+wiggle_x: 0.0
```

Comment:
I think, I would prefer a wiggle_xy_offset parameter. So, we don't need two of them, it's the same pattern as with the positions and it's more clear, that we have a relative offset here..